server/services/s2s/aws.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: "AWS Nova Sonic"
+description: "Real-time speech-to-speech service implementation using AWS Nova Sonic"
+---
+
+The `AWSNovaSonicLLMService` enables natural, real-time conversations with AWS Nova Sonic. It provides built-in audio transcription, voice activity detection, and context management for creating interactive AI experiences. It provides:
+
+<CardGroup cols={2}>
+  <Card title="Real-time Interaction" icon="bolt">
+    Stream audio in real-time with low latency response times
+  </Card>
+
+{" "}
+
+<Card title="Speech Processing" icon="waveform-lines">
+  Built-in speech-to-text and text-to-speech capabilities with multiple voice
+  options
+</Card>
+
+{" "}
+
+<Card title="Voice Activity Detection" icon="microphone">
+  Automatic detection of speech start/stop for natural conversations
+</Card>
+
+  <Card title="Context Management" icon="brain">
+    Intelligent handling of conversation history and system instructions
+  </Card>
+</CardGroup>
+
+## Installation
+
+To use `AWSNovaSonicLLMService`, install the required dependencies:
+
+```bash
+pip install "pipecat-ai[aws-nova-sonic]"
+```
+
+We recommend setting up your AWS credentials as environment variables, as you'll need them to initialize `AWSNovaSonicLLMService`:
+
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_ACCESS_KEY_ID`
+- `AWS_REGION`
+
+## Basic Usage
+
+Here’s a simple example of setting up a conversational AI bot with AWS Nova Sonic:
+
+```python
+from pipecat.services.aws_nova_sonic.aws import AWSNovaSonicLLMService
+
+llm = AWSNovaSonicLLMService(
+    secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
+    access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
+    region=os.getenv("AWS_REGION")
+    voice_id="tiffany",                    # Voices: matthew, tiffany, amy
+)
+```
+
+## Configuration
+
+### Constructor Parameters
+
+<ParamField path="secret_access_key" type="str" required>
+  Your AWS secret access key
+</ParamField>
+
+<ParamField path="access_key_id" type="str" required>
+  Your AWS access key ID
+</ParamField>
+
+<ParamField path="region" type="str" required>
+  AWS region for the service (e.g., `"us-east-1"`). Note that `"us-east-1"` is
+  the only support AWS Nova Sonic region as of 2025-05-08.
+</ParamField>
```

Comment:
Actually probably here is the right place to add that link.

Reply:
Done